### ISSUE TITLE
[beta] Icons missing in settings

### ISSUE THREAD

**Issue by Glitchy-Tozier** [florisboard/florisboard#1428]
#### Short description
This may have been intentional, but this icon is missing:
![Screenshot_20211201-133718_FlorisBoard Beta.jpg](https://user-images.githubusercontent.com/59611881/144841836-21c2ec48-66e1-46e9-b5dd-a96b913a2560.jpg)

Additionally, in some sections all of the icons are missing:
![Screenshot_20211206-125535_FlorisBoard Beta.jpg](https://user-images.githubusercontent.com/59611881/144841898-28c7f333-0db3-4f47-a353-0e4ec48d1b3b.jpg)

#### Environment information
- FlorisBoard Version: 0.3.14-beta06 (62)
- Install Source: Google PlayStore
- Device: Samsung Galaxy S9
- Android: 10

**Comment by patrickgold** [comment]
Thanks for your bug report!

Both are currently intentional, but only the second is intended for the stable 0.3.14 release.

For the first one, the Smartbar ("Schnellzugriffsleiste"), I haven't been able to find a fitting icon yet. Do you have an idea what I could use here? I am so out of ideas what a meaningful icon for the Smartbar could be...

**Comment by patrickgold** [comment]
I also re-labeled it as a proposal because it isn't really a bug IMO.

**Comment by Glitchy-Tozier** [comment]
>Do you have an idea what I could use here? I am so out of ideas what a meaningful icon for the Smartbar could be...

What settings are you planning to store there?

> I also re-labeled it as a proposal because it isn't really a bug IMO.

Yeah, I wasn't sure what to label it as. I thought maybe it was unintended, thus I chose `bug`.

**Comment by Glitchy-Tozier** [comment]
>Both are currently intentional, but only the second is intended for the stable 0.3.14 release.

Why is adding no icons intended?  🤔

**Comment by patrickgold** [comment]
> What settings are you planning to store there?

For now, these settings are available in the Smartbar screen (although I intent to add a few more customization options for the Smartbar in the future):

![image](https://user-images.githubusercontent.com/19412843/145286537-d7655f1c-b8c3-448a-a1f6-d8a288c96afb.png)

> Why is adding no icons intended? thinking

Because it is a sub-screen with just "normal" preferences. If someone suggests icons for these I have nothing against adding them, I just won't forcibly add icons just so some are there.

**Comment by Glitchy-Tozier** [comment]
> Because it is a sub-screen with just "normal" preferences. If someone suggests icons for these I have nothing against adding them, I just won't forcibly add icons just so some are there.

Personally, I think icons enourmously improve the intuitiveness of settings. I mean look how pretty this is, and how well users know what to expect, evem before reading title and subtitle:
![Screenshot_20211209-082518_FlorisBoard Beta.jpg](https://user-images.githubusercontent.com/59611881/145352553-6595ea99-0fc2-4300-b678-5fd1b41ec83b.jpg)

That being said, finding good icons takes time, and there are arguably more important things to do, like releasing the next stable version (and the one after that) as fast as possible.

**Comment by patrickgold** [comment]
> Personally, I think icons enourmously improve the intuitiveness of settings. I mean look how pretty this is, and how well users know what to expect, evem before reading title and subtitle:

I agree that icons look pretty and at some point there will be icons in this screen, but as you correctly said, this is something which is just a nice-to-have, and currently I am using all my FlorisBoard time for getting the preference and keyboard UI rework done.

**Comment by patrickgold** [comment]
Above PR adds an icon for the Smartbar from the Material icon library ([smart_button](https://fonts.google.com/icons?selected=Material%20Icons%3Asmart_button%3A)). The reason I chose that icon is because it resembles a bar with stars, and stars are commonly associated by Google with the term "smart".

Also the Advanced screen now got full icons, but some of the other sub-screens still remain iconless. Like I said above if either I or someone else has an idea which icons could be used to fill the sub-screens I gladly accept the ideas, but IMO the blank space in those screens doesn't look to bad, considering it is pretty common in apps following Material 2 standards.

This change will be included in the 0.3.14-beta12 release. Thanks again for your issue!
